layouts/partials/header.html

```diff
@@ -16,7 +16,11 @@
         <div class="nav-links">
             {{ range .Site.Menus.main }}
             <div class="nav-link">
-                <a href="{{ .URL | absURL }}" {{- if .Params.NewPage -}}target="_blank"{{- end -}}>
+	      {{ $target := "_self" }}
+	      {{ if .Params.NewPage }}
+	       {{ $target = "_blank" }}
+	      {{ end }}
+                <a href="{{ .URL | absURL }}" target="{{ $target }}">
```

Comment:
Please keep this in the same line, `target="{{ if }} ... {else} ..."`

Reply:
Also, since _self is default why do we need to change this

Reply:
Can you share a rendered HTML before and after your change ?

Reply:
Before my change, the HTML looks like this:

`<a href="https://github.com/nalysius"target="_blank"><span data-feather='github'></span>  </a>`

After my change, it looks like that:

` <a href="https://github.com/nalysius" target="_blank"><span data-feather='github'></span>  </a>`

The space before the "target" attribute was missing, and I believe it's because of the inline condition:

`<a href="{{ .URL | absURL }}" {{- if .Params.NewPage -}}target="_blank"{{- end -}}>`

That's why I used the tried to always set the target attribute, with _self or _blank. I'm not used to go template, maybe there is a better way to do it.

Reply:
Wondering if just adding a space before target works:
`<a href="{{ .URL | absURL }}" {{- if .Params.NewPage -}} target="_blank"{{- end -}}>`

Reply:
I tried this at first, but it didn't work. I guess the template engine removes spaces around "{{" and "}}".

![image](https://github.com/user-attachments/assets/d84c13b9-b605-4f24-a55a-239d2eec9abc)

![image](https://github.com/user-attachments/assets/f0856849-b5e4-4b9b-97e4-36ca4e7625a6)